library/Zend/Pdf/FileParser/Font/OpenType.php

```diff
@@ -898,7 +898,7 @@ protected function _parseCmapTable()
                     if ($language != 0) {
                         $this->_debugLog('Type 0 cmap tables must be language-independent;'
                                          . ' language: %d; skipping', $language);
-                        continue;
+                        break;
```

Comment:
This break can be omitted. 
Has anyone tested the effect of this change?

Reply:
Yes - can be removed - I tested it and create new PR:
https://github.com/magento-techdivision/zf1/pull/2